doc/tutorial/text_analytics/working_with_text_data.rst

```diff
@@ -449,7 +449,7 @@ that we can use to ``predict``::
 The object's ``best_score_`` and ``best_params_`` attributes store the best
 mean score and the parameters setting corresponding to that score::
 
-  >>> gs_clf.best_score_
+  >>> gs_clf.best_score_ # doctest: +ELLIPSIS
```

Comment:
@jnothman Fixed. Instead of two spaces, I have aligned it with rest of the existing `# doctest: +ELLIPSIS` declarations.